policyengine_us/variables/gov/states/de/tax/income/deductions/itemized/de_itemized_deductions.py

```diff
@@ -0,0 +1,23 @@
+from policyengine_us.model_api import *
+
+
+class de_itemized_deductions(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Delaware itemized deductions"
+    unit = USD
+    definition_period = YEAR
+    reference = (
+        "https://revenuefiles.delaware.gov/2022/PIT-RES_TY22_2022-02_Instructions.pdf#page=7"
+        "https://revenuefiles.delaware.gov/2022/PIT-RES_TY22_2022-02_Instructions.pdf#page=11"
+        "https://casetext.com/statute/delaware-code/title-30-state-taxes/part-ii-income-inheritance-and-estate-taxes/chapter-11-personal-income-tax/subchapter-ii-resident-individuals/section-1109-itemized-deductions-for-application-of-this-section-see-66-del-laws-c-86-section-8"
+    )
+    defined_for = StateCode.DE
+
+    def formula(tax_unit, period, parameters):
+        itm_deds_less_salt = tax_unit("itemized_deductions_less_salt", period)
+        # Self employed filers can deduct
```

Comment:
Law says only the portion they didn't deduct as part of the federal medical expense deduction